core/src/main/java/com/linecorp/armeria/server/http/HttpServerPipelineConfigurator.java

```diff
@@ -91,6 +91,7 @@ public HttpServerPipelineConfigurator(
     @Override
     protected void initChannel(Channel ch) throws Exception {
         final ChannelPipeline p = ch.pipeline();
+        p.addLast(config.maxConnectionHandler());
```

Comment:
I didn't know about `ServerBootstrap.handler()` before. I will take a look on the class you mentioned and fix the codes.